Fix dsym warnings during build macOS (#1900)
microsoft/appcenter-sdk-apple: CHANGELOG.md

# App Center SDK for iOS, macOS and tvOS Change Log

> ## Version 2.5.3

Reviewer: ```suggestion
## Version 2.5.3  (Under development)
```

### App Center

* **[Fix]** Improve log messages for errors when it failed to read/write auth token history.

> * **[Fix]** Fixed some dsym warnings which could appear on macOS in release mode.

Reviewer: ```suggestion
* **[Fix]** Fixed some dSYM warnings which could appear on macOS in release mode.
```

> * **[Fix]** Fixed some dSYM warnings which could appear on macOS in release mode.

Reviewer 2: ```suggestion
* **[Fix]** Disable module debugging for release mode in the SDK to fix dSYM warnings.
```

> * **[Fix]** Disable module debugging for release mode in the SDK to fix dSYM warnings.

Reviewer: Merge develop changes and move it to next version
